### PR TITLE
[bug 1171498] First pass on style fixes

### PR DESCRIPTION
**Pull request by willkg**
* made the text color match the face color
* reduced line-height of face description text
* tweaked font size for the aside

r?

### CODE REVIEW

File: fjord/feedback/static/css/feedback.less [hunk 1]
```diff
@@ -59,6 +59,12 @@ body.html-rtl .ask {
   text-align: right;
 }
 
+.ask-large {
+  text-align: center;
```
Comment (rlr) [inline fjord/feedback/static/css/feedback.less]: alphabetize

**Comment by rlr** [comment]
Changes LGTM. r+ w/ nit

**Comment by willkg** [comment]
^^^ Fixes the nit.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**